README.md

```diff
@@ -1595,6 +1597,207 @@ for domain in domains:
     print(domain)
 ```
 
+### Create StaticBrcodes
+
+StaticBrcodes store account information via a QR code or an image 
+that represents a PixKey and a few extra fixed parameters, such as 
+an amount and a reconciliation ID. They can easily be used to 
+receive Pix transactions.
+
+```python
+import starkinfra
+
+brcodes = starkinfra.staticbrcode.create([
+    starkinfra.StaticBrcode(
+        name="Jamie Lannister",
+        key_id="+5511988887777",
+        amount=100,
+        reconciliation_id="123",
+        city="Rio de Janeiro"
+    )
+])
+
+for brcode in brcodes:
+    print(brcode)
+```
+
+### Query StaticBrcodes
+
+You can query multiple StaticBrcodes according to filters.
+
+```python
+import starkinfra
+
+brcodes = starkinfra.staticbrcode.query(
+    limit=1,
+    after="2022-06-01",
+    before="2022-06-30",
+    uuids=["5ddde28043a245c2848b08cf315effa2"]
+)
+
+for brcode in brcodes:
+    print(brcode)
+```
+
+### Get a StaticBrcodes
+
+After its creation, information on a StaticBrcode may be retrieved by its UUID.
+
+```python
+import starkinfra
+
+brcode = starkinfra.staticbrcode.get("5ddde28043a245c2848b08cf315effa2")
+
+print(brcode)
+```
+
+### Create DynamicBrcodes
+
+Brcodes store information represented by Pix QR Codes, which are used to send or receive Pix transactions in a convenient way.
+DynamicBrcodes represent charges whose information can change at any time since all information needed for the payment is requested dynamically to an URL stored in the Brcode.
```

Comment:
DynamicBrcodes represent charges with information that can change at any time, since all data needed for the payment is requested dynamically to an URL stored in the BR Code.